src/AlgebraicGeometry/Schemes/ProjectiveAlgebraicSet/Objects/Constructors.jl

```diff
@@ -64,6 +65,15 @@ function vanishing_locus(I::MPolyIdeal{<:MPolyDecRingElem}; check::Bool=true)
   return ProjectiveAlgebraicSet(X, check=check)
```

Comment:
concerning vanishing_locus(I::MPolyIdeal{<:MPolyDecRingElem}:
How do you handle non-standard grading?

Reply:
Not at all.

Reply:
But it seems that there is a bug. It calls
`total_degree`  which does not seem to return what is expected or 
is not properly documented. 
Anyways this bug should not concern us here.

Reply:
We should explicitly require a standard grading for now and throw an 'not implemented' error otherwise.